main.py

```diff
@@ -44,13 +45,34 @@ def parse_cmd_arguments():  # travis handler, taken from https://github.com/appu
 token = config.token
 
 bot = commands.Bot(command_prefix=prefix, description=description)
-
+bot.warn_db_storage = config.warn_db_storage
 if not os.path.exists('saves/warns.json'):
     data = {}
     with open('saves/warns.json', 'w') as f:
         json.dump(data, f, indent=4)
 bot.warns_dict = json.load(open('saves/warns.json', 'r'))
-
+if bot.warn_db_storage:
+    connected = False
+    try:
+        # try connecting to the database
+        bot.db = pymongo.MongoClient(config.db_address, serverSelectionTimeoutMS=3000)

```

Comment:
I don't necessarily like config values being used midcode like this. Makes it harder to find them if you need to change the variable name.

Reply:
Ehh, good enough change.